.github/ISSUE_TEMPLATE/bug.yml

```
@@ -3,6 +3,13 @@ description: Report your bugs with the theme here!
 labels: [bug]
 body:
   - type: checkboxes
+    attributes:
+      label: Something is actually broken, I am not requesting help with my configuration.
```

Perhaps the wording should be:
```suggestion
      label: I am not requesting help with my configuration and believe something is genuinely broken.
```
Bit of a minor nit but the reordering feels more natural to read out.